considerations/minutes/technical-cab/2022-10-27.md

```diff
@@ -0,0 +1,88 @@
+**Topic(s):**
+
+- CAB processes
+- [SIP-015: Stacks Upgrade of Proof-of-Transfer and Clarity](https://github.com/stacksgov/sips/pull/95)
+
+## Meeting Notes
+
+- This was the first synchronous meeting of the CAB, so we started off with
+  brief introductions of the attendees.
+- CAB processes
+  - Confirmed tasks of the CAB:
+    1. Provide expert feedback on _Accepted_ SIPs
+    2. Vote to transition SIPs from _Accepted_ to _Recommended_ or _Rejected_
+  - How should we provide expert feedback on _Accepted_ SIPs (1)?
+    - The members of the CAB will individually comment on the PR -- no
+      coordination amongst the group is necessary for this step
+  - How should we vote to transition SIPs (2)?
+    - We will ultimately make a single decision, yes or no for each SIP.
+    - We will require a 51% quorum of CAB members to have voted, and a 51%
+      majority to determine the vote
+      - As there are 9 CAB members currently, there will be a minimum of 5
+        (`floor(9/2) + 1`) members required for a vote to count and if exactly 5
+        vote, then the threshold is 3 (`floor(5/2) + 1`)
+    - Vote will be asynchronous, via our Discord group chat
+      - For the SIP-015 vote, voting will open on Monday, Oct. 31 at 11:00 UTC
+        and close at 21:00 UTC.
+    - The decision of the CAB will be shared via an approve/reject on the SIP's
+      PR by the chairperson of the CAB
+      - PR review to include comments for a majority opinion, as well as a
+        dissenting opinion
+      - This decision and the opinion writeups will also be recorded in the
+        [sips repository](https://github.com/stacksgov/sips/tree/main/considerations/minutes/technical-cab)
+  - These processes will be added to the
+    [CAB's bylaws](https://github.com/stacksgov/sips/blob/main/considerations/technical.md#bylaws)
+  - Discussed how code changes relate to the SIP process, and the CAB's
+    responsiblity
+    - The implementation / code supporting a SIP is typically a later step in
+      the process
+    - Evaluating the implementation of the SIP is outside the purview of this
+      CAB (though its members are appropriate candidates for code reviews)
+  - Discussed the need for a shared email list for the group, as well as a
+    mechanism for scheduling meetings
+  - Discussed an intent to setup a monthly recurring meeting for the group to
+    sync up
+    - Proposed Mondays, but will use some tool to help figure out best
+      availability for the group
+    - Additional meetings outside of this regularly scheduled meeting may be
+      required for specific SIPs and will be scheduled ad-hoc
+- SIP-015 discussions
+  - Concerns have been expressed that this SIP is too large and contains
+    unrelated items
+    - Members feel that the CAB should be able to vote on individual, related
+      items, not one kitchen sink SIP
+    - With the timing and current state of this SIP, it would be difficult to
+      make a change like this at this time
```

Comment:
Vague to say "timing and current state of this SIP" when the reason for the reluctance is that the SIP is already implemented. Change to _As the SIP is already implemented, it would be difficult to make a substantial change like this_.

Reply:
Sounds good.